src/Directory.Build.props

```diff
@@ -4,6 +4,7 @@
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <Version>4.0.1</Version>
+    <AnalysisLevel>5</AnalysisLevel>
```

Comment:
https://devblogs.microsoft.com/dotnet/automatically-find-latent-bugs-in-your-code-with-net-5/